src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,23 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private static final String START_WORK_MESSAGE = "Excavator started to work";
+
+    private int bucketCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). Consider changing the visibility of `bucketCapacity` to private.